Aquality.Selenium/src/Aquality.Selenium/Browsers/IBrowserTabNavigation.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections.Generic;
+
+namespace Aquality.Selenium.Browsers
+{
+    /// <summary>
+    /// Provides functionality to work with browser tab navigation.  
+    /// </summary>
+    public interface IBrowserTabNavigation
+    {
+        /// <summary>
+        /// Switches to tab.
+        /// </summary>
+        /// <param name="name">Tab handle.</param>
+        /// <param name="closeCurrent">Close current tab if true and leave it otherwise.</param>
+        void SwitchToTab(string tabHandle, bool closeCurrent = false);
+
+        /// <summary>
+        /// Switches to tab.
+        /// </summary>
+        /// <param name="index">Tab index.</param>
+        /// <param name="closeCurrent">Close current tab if true and leave it otherwise.</param>
+        void SwitchToTab(int index, bool closeCurrent = false);
+
+        /// <summary>
+        /// Switches to new tab.
```

Comment:
Should this method create this new tab? If so, please add it to summary.

Reply:
Nope, I supposed method will just switch to opened tab.
If user want to open new, he can use specified method

Reply:
But the opened tab not necessary the new one. So, does it make sense to keep this method in interface?
It seems that the name should be:
void SwitchToTab(bool closeCurrent = false);

Reply:
got you, fixed